tools/packaging/kata-deploy/README.md

```diff
@@ -280,3 +280,54 @@ This DaemonSet runs of the node has the label `katacontainers.io/kata-runtime=cl
 the `katacontainers.io/kata-runtime` label as well as restarts either CRI-O or `containerd` `systemctl`
 daemon. You cannot execute these resets during the `preStopHook` of the Kata installer DaemonSet,
 which necessitated this final cleanup DaemonSet.
+
+## Development
+
+This section is for developers who need to fix, extend and/or implement features to Kata deploy.
+
+### Adding a new runtimeClass
+
+Kata deploy installs and configure [Kubernetes runtime classes](https://kubernetes.io/docs/concepts/containers/runtime-class/)
+matching the several Kata Containers [runtime configuration files](../../../src/runtime/config/).
+
+The deployed runtimeClass yaml files are hosted at [`runtimeclasses`](./runtimeclasses/) directory. In particular,
+[`kata-runtimeClasses.yaml`](./runtimeclasses/kata-runtimeClasses.yaml) isn't properly a runtimeClass file but rather
+its content is just the concatenation of all other yaml files in the directory.
+
+If you need to add a new runtimeClass then do:
+
+ * Create the runtimeClass yaml file in [`runtimeclasses`](./runtimeclasses/). Follows the name convention of `kata-<MY-RUNTIME-CLASS-NAME>.yaml`.
+ * Update [`kata-runtimeClasses.yaml`](./runtimeclasses/kata-runtimeClasses.yaml). Notice that the entries are sorted by runtimeClass name.
```

Comment:
I'm tempted to suggest using the `check-runtimeclasses.sh` to generate this file and simply copy the `resultingRuntimeClasses`, what do you think?

Reply:
Of course with a big fat warning asking the developer to review the diff :-)

Reply:
I'm not sure. I prefer to keep as is.

Reply:
Sure, no problem